pages/common/vala.md

```diff
@@ -0,0 +1,17 @@
+# vala
+
+> Vala code runner.
+> Tutorial: <https://wiki.gnome.org/Projects/Vala/Tutorial>.
+> Doc homepage: <https://valadoc.org/>.
```

Comment:
The tutorial seems to be about Vala the language, not the vala command itself. That said, I can't seem to find any documentation about `valac` directly. We generally prefer just 1 link here, so what about this?
```suggestion
> More information: <https://valadoc.org>.
```